pkg/workloads/console/controller.go

```diff
@@ -78,6 +82,54 @@ func (r *ConsoleReconciler) Reconcile(request reconcile.Request) (res reconcile.
 		return res, err
 	}
 
-	logger.Log("event", "Reconciled") // TODO
-	return
+	logger.Log("msg", "found console")
```

Comment:
Can we get this logging events, and using `logging.WithRecorder`? That way we can debug our controllers by describing the consoles which would be rad. The directoryrolebindings give a good example of this: it's probably clearer to create event constants ([link](https://github.com/lawrencejones/theatre/blob/master/pkg/rbac/directoryrolebinding/controller.go#L30-L38)) and log those than messages, but no harm keeping the messages if we think they're useful.